orchestrator/crates/prover-clients/atlantic-service/src/lib.rs

```diff
@@ -178,63 +211,123 @@ impl ProverClient for AtlanticProverService {
 
         Ok(atlantic_job_response.atlantic_query_id)
     }
+
+    async fn get_aggregator_task_id(
+        &self,
+        bucket_id: &str,
+        aggregator_index: u64,
+    ) -> Result<String, ProverClientError> {
+        let bucket = self.atlantic_client.get_bucket(bucket_id).await?;
+
+        Ok(bucket
+            .queries
+            .iter()
+            .find(|query| match query.bucket_job_index {
+                Some(index) => index == aggregator_index,
+                None => false,
+            })
+            .ok_or(ProverClientError::FailedToGetAggregatorId(bucket_id.to_string()))?
+            .id
+            .clone())
+    }
+
+    async fn get_task_artifacts(
+        &self,
+        task_id: &str,
+        task_type: TaskType,
+        file_name: &str,
+    ) -> Result<Vec<u8>, ProverClientError> {
+        match task_type {
+            TaskType::Query => Ok(self
+                .atlantic_client
+                .get_artifacts(format!("{}/queries/{}/{}", ATLANTIC_FETCH_ARTIFACTS_BASE_URL, task_id, file_name))
+                .await?),
+            TaskType::Bucket => Ok(self
+                .atlantic_client
+                .get_artifacts(format!("{}/queries/{}/{}", ATLANTIC_FETCH_ARTIFACTS_BASE_URL, task_id, file_name))
+                .await?),
+        }
+    }
```

Comment:
1. what is task artifacts?
2. I can simply call the endpoint without checking the task_type right?
```suggestion
    async fn get_task_artifacts(
        &self,
        task_id: &str,
        task_type: TaskType,
        file_name: &str,
    ) -> Result<Vec<u8>, ProverClientError> {
           Ok(self
                .atlantic_client
                .get_artifacts(format!("{}/queries/{}/{}", ATLANTIC_FETCH_ARTIFACTS_BASE_URL, task_id, file_name))
                .await?)
    
    }
```

Reply:
1. cairo pie, proof, snos output, metadata, etc.
2. Yes, we can do that. For now, the path for both are same, but kept this just in case it differs in the future
Removed this now